packages/recommend/src/methods/getRecommendations.ts

```diff
@@ -0,0 +1,44 @@
+import { SearchOptions } from '@algolia/client-search';
+import { MethodEnum } from '@algolia/requester-common';
+
+import { RecommendClient, RecommendModel, WithRecommendMethods } from '../types';
+
+export type GetRecommendationsOptions = {
+  readonly indexName: string;
+  readonly model: RecommendModel;
+  readonly objectID: string;
+  readonly threshold?: number;
+  readonly maxRecommendations?: number;
+  readonly queryParameters?: SearchOptions;
+  readonly fallbackParameters?: Pick<SearchOptions, 'facetFilters'>;
```

Comment:
Shouldn't they both have the same type?

```ts
type GetRecommendationsSearchParameters  = Omit<
  SearchOptions,
  'page' | 'hitsPerPage' | 'offset' | 'length'
>;
```

Reply:
Good call! See 911c7db50f125738fe741b11591cc0e64d4e67a8.